models/user/index.js

```diff
@@ -414,6 +414,37 @@ class User extends Model {
     return output;
   }
 
+
+    /**
+   * Find [limit] new users (newlycreated and veryfied)
+   * who has at least [similarTagsNumber] similar tags to mine
+   * @param {int, int} limit, similatTagsNumber
+   * @returns {Promise<Object>}
+   */
+  static async findNewUsersWithMyTags(myUsername, limit, commonTagsNumber) {
+    const query = `
+    FOR similarUser in (
+    FOR u IN users
+    FILTER u.username == @myUsername
+    FILTER TO_BOOL(u.email) == true
+    FOR v,e,p IN 2 OUTBOUND u
+      ANY userTag
```

Comment:
here might follow `FILTER TO_BOOL(v.email) == true` instead of `FILTER TO_BOOL(similarUser.foundUser.email) == true` being below. Might be more efficient (filtering out unconfirmed users immediately).
But perhaps it doesn't matter because unconfirmed users aren't able to add `userTag` relations.